Update CHANGELOG.md for MSAL Release 4.61.1 (#4782)
AzureAD/microsoft-authentication-library-for-dotnet: CHANGELOG.md

4.61.1
==========
### New Features
- Enabled Proof of Possession (POP) extensibility API for all confidential client requests, enhancing security measures for confidential clients. See [4757](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4757)

> - Introduced a public API to detect and return the current environment for Managed Identity. This API helps identify and utilize the appropriate managed identity source, streamlining the management of different environments. See [4751](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4751)

Reviewer: ```suggestion
- Introduced a public API that identifies and returns the current Azure environment for Managed Identity. See [4751](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4751)
```
